app/qml/main.qml

```diff
@@ -330,6 +344,32 @@ ApplicationWindow {
       }
     }
 
+    NavigationPanel {
```

Comment:
I was thinking about having this entire panel inside `MapWrapper.qml` and reacting on `_map` state changes. In that case you would not need to set different components here in `main.qml` to visible/invisible. I am happy to discuss that over call @NEDJIMAbelgacem